src_c/base.c

```diff
@@ -1124,10 +1122,7 @@ pgObject_GetBuffer(PyObject *obj, pg_buffer *pg_view_p, int flags)
             return -1;
         }
         success = 1;
-    }
-
-#endif
-    if (!success && pgGetArrayStruct(obj, &cobj, &inter_p) == 0) {
+    }    if (!success && pgGetArrayStruct(obj, &cobj, &inter_p) == 0) {
```

Comment:
You deleted one linebreak too many, now it looks like an else if branch.

Reply:
Good catch - fix is in.